nym-wallet/src/components/Accounts/MultiAccountWithPwdHowTo.tsx

```diff
@@ -22,10 +22,10 @@ export const MultiAccountWithPwdHowTo = ({ show, handleClose }: { show: boolean;
   >
     <Stack spacing={2}>
       <Warning sx={{ textAlign: 'center' }}>
-        <Typography fontWeight={600} sx={{ mb: 1 }}>
+        <Typography fontWeight={600} fontSize={14} sx={{ mb: 1 }}>
           This machine already has a password set on it
         </Typography>
-        <Typography>
+        <Typography fontSize={12}>
```

Comment:
can we use variant here to achieve the correct fontsize? I believe the value body2 might work